src/arguments/CoreInteger.ts

```diff
@@ -1,41 +1,27 @@
 import type { PieceContext } from '@sapphire/pieces';
 import { Identifiers } from '../lib/errors/Identifiers';
+import { resolveInteger } from '../lib/resolvers';
 import { Argument, ArgumentContext, ArgumentResult } from '../lib/structures/Argument';
 
 export class CoreArgument extends Argument<number> {
+	private readonly messages = {
+		[Identifiers.ArgumentIntegerTooSmall]: ({ minimum }: ArgumentContext) => `The argument must be greater than ${minimum}.`,
+		[Identifiers.ArgumentIntegerTooBig]: ({ maximum }: ArgumentContext) => `The argument must be less than ${maximum}.`,
+		[Identifiers.ArgumentIntegerError]: () => 'The argument did not resolve to a valid integer.'
```

Comment:
```suggestion
		[Identifiers.ArgumentIntegerTooSmall]: ({ minimum }: ArgumentContext) => `The given number must be greater than ${minimum}.`,
		[Identifiers.ArgumentIntegerTooLarge]: ({ maximum }: ArgumentContext) => `The given number must be less than ${maximum}.`,
		[Identifiers.ArgumentIntegerError]: () => 'The argument did not resolve to a valid number.'
```

And update `resolveInteger` accordingly